### PR TITLE
chore: fix enable static website progress bar

### PR DESCRIPTION
**Pull request by Siglud**
*  Fix progress bar count error
*  Add retry when upload to Azure encounter server-side error
*  Fix error JS/TS project enable WEBSITE_RUN_FROM_PACKAGE will cause deploy error

### CODE REVIEW

File: templates/scenarios/ts/workflow/infra/azure.bicep [hunk 1]
```diff
@@ -48,17 +48,18 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
...
-        }
+        } */
         {
           name: 'RUNNING_ON_AZURE'
           value: '1'
         }
         {
           name: 'BOT_ID'
-          value: botAadAppClientId
+          value: botAadAppClientIdl
```
Comment (hund030) [inline templates/scenarios/ts/workflow/infra/azure.bicep:62]: Seems this value is wrong?

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -111,6 +111,10 @@
   "core.progress.uploadZipFileToAzure": "Uploading application package to Azure.",
   "core.progress.checkAzureDeployStatus": "Waiting for Azure deployment to complete.",
   "core.progress.restartAzureFunctionApp": "Restarting Azure Function App.",
+  "core.progress.configureAzureStorage": "Confining Azure Storage, enable static website setting.",
```
Comment (IvanJobs) [inline packages/fx-core/resource/package.nls.json:114]: Configuring?

File: templates/scenarios/ts/workflow/infra/azure.bicep (hunk 1 above)
Comment (eriolchan) [inline templates/scenarios/ts/workflow/infra/azure.bicep]: Why is this value "botAadAppClientIdl"?

Comment (Siglud) [reply templates/scenarios/ts/workflow/infra/azure.bicep]: Yes! A slip of the pen. fixed

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (Siglud) [reply packages/fx-core/resource/package.nls.json]: fixed

**Review by eriolchan — DISMISSED** [review]

**Review by hund030 — DISMISSED** [review]

File: packages/vscode-extension/.vscode/launch.json [hunk 3]
```diff
@@ -16,7 +16,9 @@
         "${workspaceFolder}/../api/build/**/*.js"
       ],
       "env": {
-        "NODE_ENV": "development"
+        "NODE_ENV": "development",
+        "TEAMSFX_V3": "true",
+        "TEAMSFX_DEBUG_TEMPLATE": "true"
```
Comment (hund030) [inline packages/vscode-extension/.vscode/launch.json:21]: Maybe revoke this chagne.

Comment (Siglud) [reply packages/vscode-extension/.vscode/launch.json]: Fixed

**Review by hund030 — DISMISSED** [review]

**Review by eriolchan — APPROVED** [review]

**Pull request merged**